### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by locks**
Fixes https://github.com/glimmerjs/glimmer-application/issues/84.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,11 +12,10 @@ module.exports = {
   // ],
 
   locals(options) {
-    let name = options.entity.name;
-    let component = componentize(name);
+    let name = componentize(options.entity.name);
```
Comment (rwjblue) [inline index.js]: `componentize` adds `-app` if the name of the project was only a single word, we don't need that here.

Instead we should do:

```js
let name = dasherize(options.entity.name);
let component = componentize(name);
let className = stringUtils.classify(options.entity.name);
return { name, className, component };
```

Comment (rwjblue) [inline index.js]: And then we need to update `files/config/environment.js` to use `name` instead of `component` for the `modulePrefix`

Comment (locks) [inline index.js]: Gracias, I will update :)
We… don't test the blueprint, do we?

Comment (rwjblue) [inline index.js]: There are no tests in this repo, no.  I generally test locally while making changes with `ember new foo -b ../path/to/this/repo`

**Comment by locks** [comment]
@rwjblue updated and tested manually 👍

**Pull request merged**